testing/assertions_skia.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TESTING_ASSERTIONS_SKIA_H_
+#define FLUTTER_TESTING_ASSERTIONS_SKIA_H_
+
+#include "flutter/testing/assertions.h"
+
+namespace flutter {
+namespace testing {
+
+//
```

Comment:
Do we need this?

Reply:
No. It was for assertions that were later moved to `embedder_assertions.h`. Removed.